lib/auth/auth.go

```diff
@@ -1268,10 +1265,7 @@ func (a *Server) deleteMFADeviceSafely(ctx context.Context, user, deviceName str
 			return trace.BadParameter(
 				"cannot delete the last MFA device for this user; add a replacement device first to avoid getting locked out")
 		}
-	case constants.SecondFactorWebauthn:
-		// TODO(codingllama): Handle webauthn device deletion.
-		return trace.NotImplemented("webauthn device deletion not yet implemented")
-	default:
+	default: // SecondFactorOTP, SecondFactorU2F, SecondFactorWebauthn
```

Comment:
would it be best to have an explicit `case SecondFactorOTP, SecondFactorU2F, SecondFactorWebauthn:` and then leave the default case as an error case?

My thinking is this would prevent a case where we add a new second factor type and it mistakenly falls into the default case here.

Reply:
Done.